pyproject.toml

```
@@ -60,9 +60,9 @@ dependencies = [
     "jsonlines>=4.0.0",
     "numba>=0.59.0",
     "omegaconf>=2.3.0",
-    "opencv-python>=4.9.0",
+    "opencv-python-headless>=4.9.0",
     "packaging>=24.2",
-    "av>=12.0.5,<13.0.0",
+    "av>=14.0.0",
```

I think we can keep this lower bound
```suggestion
    "av>=12.0.5",
```

---

Nope. If by any chance `av` get's resolves to `12.X`, it would require `ffmpeg6` leading us to incompatibility issues with the conda-forge `ffmpeg7` installed

---

After reviewing, I see that **ffmpeg6** is only required when building **av** (as we previously did with `--no-binary`). Since we’ve already removed that, I agree that keeping it with **12.X** should be safe. That said, the dependency manager will ultimately resolve it to **14.X** anyway.

---

The point of having lower dependencies constraints is to allow greater compatibility with lerobot as a dependency. If some end-user project already has a contraint such as `av<13`, they wouldn't be able to also have lerobot as a dependency whereas they can with the lower constraint.